packages/analytics-core/src/analytics.js

```diff
@@ -60,6 +60,11 @@ export default class AvAnalytics {
 
     this.plugins = Array.isArray(plugins) ? plugins : [plugins];
     this.pageTracking = !!pageTracking;
+    this.eventModifiers = options.eventModifiers
+      ? Array.isArray(options.eventModifiers)
```

Comment:
Yeah so let's change this to use `if` statement with ternary vs our nested ones. Seems like we have a linter setting for it.

Reply:
on it